Detections/MultipleDataSources/RiskyUserIn3Pnetworkactivity.yaml

```diff
@@ -0,0 +1,91 @@
+id: 042f2801-a375-4cfd-bd29-041fc7ed88a0
+name: Risky User observed in non-microsoft network device 
```

Comment:
Capital M on Microsoft.

Also should this say "Risky user signin" rather than just "Risky user"?